solidity/contracts/bridge/Bridge.sol

```diff
@@ -1460,68 +1474,38 @@ contract Bridge is Ownable {
 
     // TODO: Function `submitRedemptionFraudProof`. That function must:
     //       1. Take a `BitcoinTx.Info` and `BitcoinTx.Proof` of the
-    //          fraudulent transaction. It should also accept `walletPubKeyHash`.
+    //          fraudulent transaction. It should also accept `walletPubKeyHash`
+    //          and index of fraudulent output.
```

Comment:
Index of fraudulent input as well I think. There could be a transaction with 1000 inputs, we do not want to iterate over them and look at just one problematic  input pointed by the reported. Think of a dozen valid deposit reveal inputs during a sweep.

Reply:
See https://github.com/keep-network/tbtc-v2/pull/128/commits/64395ffafef55a2795e927a1dad3c531d0ef24d8